RELEASE-NOTES.md

```diff
@@ -3,6 +3,35 @@ id: Release notes
 section: developer-resources
 releaseNoteTOC: true
 ---
+## 2022.08 release notes (2022-06-24)
+Packages released:
+- [@patternfly/patternfly@v4.202.1](https://www.npmjs.com/package/@patternfly/patternfly/v/4.202.1)
+
+### Components
+- **Card:**
+  - Added styling to cards when their hidden input is focused ([#4902](https://github.com/patternfly/patternfly/pull/4902))
+  - Fixed dark theme card, light bg contrast ([#4903](https://github.com/patternfly/patternfly/pull/4903))
+  - Forced dark theme dark section card bg color ([#4913](https://github.com/patternfly/patternfly/pull/4913))
+- **Description list:** Added display and card mods ([#4895](https://github.com/patternfly/patternfly/pull/4895))
+- **Drawer:** Removed duplicate body element on jumplinks demo ([#4910](https://github.com/patternfly/patternfly/pull/4910))
+- **Dropdown:** Added secondary split toggle ([#4897](https://github.com/patternfly/patternfly/pull/4897))
+- **Masthead:** Fixed default/dynamic viewport based inset when using resize observer ([#4919](https://github.com/patternfly/patternfly/pull/4919))
+- **Menu:** Fixed scrollbar on flyout variant ([#4892](https://github.com/patternfly/patternfly/pull/4892))
+- **Menu toggle:** Updated typeahead variant layout/spacing ([#4750](https://github.com/patternfly/patternfly/pull/4750))
+- **Notification badge:** Fixed documentation links ([#4891](https://github.com/patternfly/patternfly/pull/4891))
+- **Progress stepper:**
+  - Added horizontal/vertical breakpoint support ([#4901](https://github.com/patternfly/patternfly/pull/4901))
+  - Used button for step title help text ([#4912](https://github.com/patternfly/patternfly/pull/4912))
+- **Table:**
+  - Added tr border to sticky nested table header ([#4857](https://github.com/patternfly/patternfly/pull/4857))
+  - Removed bullseye layout in demos ([#4904](https://github.com/patternfly/patternfly/pull/4904))
+- **Tabs:** Added expandable overflow styling ([#4876](https://github.com/patternfly/patternfly/pull/4876))
+- **Page:** Added height breakpoints for sticky top and bottom modifiers ([#4905](https://github.com/patternfly/patternfly/pull/4905))
+- **Wizard:** Moved footer to sibling of drawer ([#4896](https://github.com/patternfly/patternfly/pull/4896))
+
+### Other
+- **Build:** Copied theme scss files to dist ([#4878](https://github.com/patternfly/patternfly/pull/4878))
```

Comment:
@mcoker it looks like this went out with the last release
- https://github.com/patternfly/patternfly/pull/4878#issuecomment-1145092717
- https://www.patternfly.org/v4/developer-resources/release-notes/html#other

Reply:
good catch! I double check the PR #'s for each release in the release notes to make sure there are no dupes, but I added #4878 to the release notes manually last release and accidentally added it using the issue # for the URL (#4877) instead of the PR # so I didn't catch the dupe in this release.